test/weet.test.js

```diff
@@ -0,0 +1,9 @@
+const weetService = require('../app/services/weet');
+
+describe('Witter should return', () => {
+  it('A phrase when calling the service method', async () => {
+    const weet = await weetService.getRandomWeet();
+
+    expect(weet).not.toBeUndefined();
+  });
+});
```

Comment:
There is no need for a test right now, but if you want to test this, you should mock the external service request

Reply:
Thank you, I will keep this in mind when making the change.